content/sensu-go/5.21/reference/agent.md

```diff
@@ -715,7 +715,15 @@ Specify the agent configuration with either a `.yml` file or `sensu-agent start`
 Configuration via command line flags overrides attributes specified in a configuration file.
 See the [Example Sensu agent configuration file][5] for flags and defaults.
 
-#### Configuration summary
+### Certificate bundles or chains
+
+The Sensu backend supports all types of certificate bundles (or chains) as long as the server (or leaf) certificate is the *first* certificate in the bundle.
+This is because the Go standard library assumes that the first certificate listed in the PEM file is the server certificate &mdash; the certificate that the program will use to show its own identity.
```

Comment:
```suggestion
This is because the Go standard library assumes that the first certificate listed in the PEM file is the leaf certificate &mdash; the certificate that the program will use to show its own identity.
```